testbed/bin/testbed.ts

```diff
@@ -9,5 +9,5 @@ new TestbedStack(app, 'TestbedStack', {
         region: process.env.CDK_DEFAULT_REGION
     },
     repositories: [
-        { name: "testbed", url: "https://github.com/awslabs/kubernetes-iteration-toolkit", path: "./testbed/addons" }]
+        { name: "testbed", url: "https://github.com/awslabs/kubernetes-iteration-toolkit", path: "./testbed/addons", branch: "testbed" }]
```

Comment:
Hmm -- why do we want to store this on a branch called "testbed"?

Reply:
Like mentioned in the description, it's easier for me to test end to end (using flux)  before creating a PR. Once it's fully ready for users to use it, I will point it to HEAD on main.

Reply:
I will do a CDK deploy of this change locally as workaround.